dmcaf/condition_generator.py

```diff
@@ -117,7 +117,7 @@ def _generate_count_prompts(self, experiment_id: str, count: int) -> List[Dict]:
             number = random.choice(list(range(1, 5)))
             obj_singular = random.choice(selected_objects)
             obj_display = self.singular_to_plural[obj_singular] if number > 1 else obj_singular
-            prompt = f"Zoomed out photo of {number} {obj_display} centered in front of unicolor background"
+            prompt = f"{number} {obj_display} in front of unicolor background"
```

Comment:
what do you think of?
```suggestion
            prompt = f"Photo of {number} {obj_display} in front of unicolor background"
```